app/res/xml/preferences_developer.xml

```diff
@@ -85,6 +85,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-auto-purge"
         android:title="Detect and Trigger Purge on Form Save"/>
+    <EditTextPreference
+        android:defaultValue=""
+        android:enabled="true"
+        android:key="cc-form-record-file-path"
+        android:title="Form Record File"/>
```

Comment:
Ah, I just realized you were copying this naming pattern from what Philip was using for the remote version, which makes sense. I think that one is confusing too, and would be in favor of renaming the preference title for that similarly (something like `"Form Payload Server"`)